pkg/beacon/beacon.go

```diff
@@ -102,6 +104,16 @@ func Initialize(
 					request.BlockNumber,
 				)
 			}()
+		} else {
+			groupPublicKey, err := altbn128.UnmarshalG2Point(request.GroupPublicKey)
+			if err != nil {
+				logger.Warningf("could not relay messages: [%v]", err)
+			} else {
+				// Channel name used for relay entry request is the string
+				// representation of the compressed group public key G2 point.
+				name := hex.EncodeToString(groupPublicKey.Compress())
+				go node.RelayMessagesForChannel(name)
```

Comment:
Thoughts on naming it `ForwardSignatureShares`?

It's just a name but we'll keep the "channel" abstraction lower. What matters here is that if you are not in the group, you can still help by forwarding signature shares if needed. I was also considering `RelaySignatureShares` but since we have `OnRelayEntryRequested` and `GenerateRelayEntry`, I think `Forward*` could be less confusing.